packages/legend-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -232,12 +233,18 @@ const ProjectDependencyEditor = observer(
     // version
     const version = projectDependency.versionId;
     const versions = selectedProject?.versions ?? [];
-    const versionOptions = versions
+    let versionOptions = versions
       .slice()
       .sort((v1, v2) => compareSemVerVersions(v2, v1))
       .map((v) => ({ value: v, label: v }));
+
+    versionOptions = [
+      { label: 'HEAD', value: SNAPSHOT_VERSION_ALIAS },
```

Comment:
Don't we have the word `HEAD` as a constant somewhere? If we don't should probably live in legend-sdlc and be used here instead of writing it directly.

Reply:
We could use `SNAPSHOT_VERSION_ALIAS` from `legend-server-depot` here